server/tools.js

```diff
@@ -17,6 +17,7 @@
 module.exports = function () {
     var fs = require('fs');
     var path = require('path');
+    var multer = require('multer');
```

Comment:
The lib multer has to be added in package.json, to do it run the command

```npm install multer --save``` where ```--save``` means to add automatically multer into package.json file.